ui/components/multichain/pages/send/send.js

```diff
@@ -192,7 +193,6 @@ export const SendPage = () => {
 
     setIsSubmitting(true);
     await dispatch(signTransaction(history));
```

Comment:
Does the page fail gracefully if signing a swap + send transaction/approval fails? Example concerns: page getting stuck in loading state, user doesn't get feedback about the failed signing

Maybe `history.push(DEFAULT_ROUTE);` in signTransaction should happen in a finally block?